specification/offazurespringboot/resource-manager/Microsoft.OffAzureSpringBoot/preview/2023-01-01-preview/springbootdiscovery.json

```diff
@@ -1606,6 +1703,13 @@
             "description": "The error",
             "$ref": "#/definitions/Error"
           }
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
         }
       },
       "description": "The springbootapps resource definition."
```

Comment:
>readOnly [](http://example.com/codeflow?start=11&length=8)

Breaking change

Reply:
The provisionState should be readonly by the validation rules